### PR TITLE
Refresh the guide to match Eclipse Microprofile

### PR DESCRIPTION
**Pull request by SueChaplain**
- Introduce edits from the Eclipse MicroProfile guide for
Spring.
- Add some images to improve the guide.
- Make updates for version and Appsody hub name

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

**Comment by SueChaplain** [comment]
Hi @Charlotte-Holt - I went through the review edits on the MicroProfile guide and applied those changes here. Please can you check the guide over to see if you can spot any problems. 

I don't think we need this reviewed again by Alan or Nate, as they will be happy if this guide mirrors the MicroProfile one.

File: README.adoc [hunk 1]
```diff
@@ -189,36 +209,32 @@ Run the following command to start the Appsody development environment:
...
+
 [source, role='no_copy']
 ----
 [Container] 2019-09-12 17:28:44.066  INFO 171 --- [  restartedMain] o.s.b.a.e.web.EndpointLinksResolver      : Exposing 4 endpoint(s) beneath base path '/actuator'
 [Container] 2019-09-12 17:28:44.205  INFO 171 --- [  restartedMain] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 8080 (http) with context path ''
 [Container] 2019-09-12 17:28:44.209  INFO 171 --- [  restartedMain] application.Main                         : Started Main in 6.051 seconds (JVM running for 6.923)
 ----
 
-This message indicates that the Tomcat server is started and you are ready to begin development.
+This message indicates that the Tomcat server is started. Browser to http://localhost:8080 and you can see the Appsody splashscreen.
```
Comment (Charlotte-Holt) [inline README.adoc]: Should "splashscreen" be "splash screen"?

**Review by Charlotte-Holt — APPROVED** [review]
@SueChaplain Looks good! I just made one comment that I think might be a typo. I'll try to go into the MP guide tomorrow and make your suggested updates as well. Thanks!!

File: README.adoc (hunk 1 above)
Comment (SueChaplain) [reply README.adoc]: Yes, agreed. Just spotted a typo too ... browser / browse. Will fix up.

Thank you!

**Comment by SueChaplain** [comment]
@Charlotte-Holt - would you be able to merge these changes please? Nate/Alan don't need to re-review if the guide matches those edits requested for MicroProfile. Thanks!

**Comment by Charlotte-Holt** [comment]
@SueChaplain Yep, done.

**Pull request merged**